src/guide/lists.md

```diff
@@ -17,10 +17,10 @@ let strings = ["foo", "bar", "baz"]
 Lists in Grain are linked lists, so if we'd like to add a new item to a list, we add it to the front:
 
 ```grain
-let numbers = [2, 3]
-let numbers = [1, ...numbers]
+let pair = [2, 3]
+let triple = [1, ...pair]
```

Comment:
Shadowing `numbers` here is pretty idiomatic Grain (and just FP in general), but I could see the case for using two different names here. I think the problem with `pair` and `triple` though is that those terms are typically used with tuples, and for people coming from a language like JS, they might think this is the appropriate way to represent a pair or a triple.

Maybe something like this?

```suggestion
let twoThree = [2, 3]
let oneTwoThree = [1, ...twoThree]
```